depmapomics/dm_omics.py

```diff
@@ -760,145 +746,126 @@ async def mutationPostProcessing(
     ].replace({SAMPLEID: renaming_dict})
 
     # merge
-    print("merging")
+    print("merging WES and WGS")
     folder = WORKING_DIR + samplesetname + "/merged_"
     mergedmutations = wgsmutations.append(wesmutations).reset_index(drop=True)
+    # some hgnc symbols in the maf are outdated, we are renaming them here and then dropping ones that aren't in biomart
```

Comment:
maybe have a task to update them to the same HGNC version as what is used by ensembl v102.

Reply:
actually need to update funcotator's ref files (check gencode and hgnc version)